docs/COMMON_ACTIONS.md

```diff
@@ -0,0 +1,20 @@
+
+This example highlights using OCS with a time-series based key. All of the principals and actions shown here apply to time-series keyed streams as well. PLease see the [Time-Series](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/SDS_TIME_SERIES_README.md) readme for a complete listing of operations shown.
+
+| Tasks                                                                                              | Languages                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Test Status                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
same comment about the number of spaces. `| Tasks | Languages | Test Status |` is a lot more readable in markdown